clients/admin-ui/src/features/privacy-requests/configuration/TwilioEmailConfiguration.tsx

```diff
@@ -0,0 +1,165 @@
+import { Box, Button, Divider, Heading, Stack } from "@fidesui/react";
+import { Form, Formik } from "formik";
+import { useState } from "react";
+
+import { CustomTextInput } from "~/features/common/form/inputs";
+import { getErrorMessage } from "~/features/common/helpers";
+import { useAlert } from "~/features/common/hooks";
+import {
+  useCreateMessagingConfigurationMutation,
+  useCreateMessagingConfigurationSecretsMutation,
+  useGetMessagingConfigurationDetailsQuery,
+} from "~/features/privacy-requests/privacy-requests.slice";
+
+const TwilioEmailConfiguration = () => {
+  const [configurationStep, setConfigurationStep] = useState("");
+  const { errorAlert, successAlert } = useAlert();
+  const { data: messagingDetails } = useGetMessagingConfigurationDetailsQuery({
+    type: "twilio_email",
+  });
+  const [createMessagingConfiguration] =
+    useCreateMessagingConfigurationMutation();
+  const [createMessagingConfigurationSecrets] =
+    useCreateMessagingConfigurationSecretsMutation();
+
+  const handleTwilioEmailConfiguration = async (value: { email: string }) => {
+    const payload = await createMessagingConfiguration({
+      type: "twilio_email",
+      details: {
+        twilio_email_from: value.email,
+      },
+    });
+
+    if ("error" in payload) {
+      errorAlert(
+        getErrorMessage(payload.error),
+        `Configuring messaging provider has failed to save due to the following:`
+      );
+    } else {
+      successAlert(`Configure messaging provider saved successfully.`);
+      setConfigurationStep("2");
```

Comment:
Use numbers if the configuration steps are going to be numbers. Or they could be strings/enums, in which case they should have meaningful names. The [config wizard](https://github.com/ethyca/fides/blob/9f993379a30a92ba9995348b5ca32ecda200582e/clients/admin-ui/src/features/config-wizard/constants.tsx#L1) goes with a bit of a hybrid where there's an array of objects that could have extended info, but that could be overblown.